packages/connect/src/api/bleUnpair.ts

```diff
@@ -0,0 +1,29 @@
+import { Assert } from '@trezor/schema-utils';
+
+import { PROTO } from '../constants';
+import { AbstractMethod } from '../core/AbstractMethod';
+import { UI } from '../events';
+
+export default class BleUnpair extends AbstractMethod<'bleUnpair', PROTO.BleUnpair> {
+    init() {
+        this.allowDeviceMode = [UI.INITIALIZE, UI.SEEDLESS];
+        this.requiredPermissions = ['management'];
+        this.useDeviceState = false;
+        this.skipFinalReload = true;
```

Comment:
I see, it results into a disconnect device so no features reload is possible right?